### ISSUE TITLE
Keyboard navigation for the multiple IPs

### ISSUE THREAD

**Issue by momesgin** [rancher/dashboard#15767]
**Setup**
- Rancher version: 2.13
- Rancher UI Extensions:
- Browser type & version:

**Describe the bug**

Keyboard navigation is missing for the mulitple IPs in the table. the `+{n} more` badge should also support keyboard navigation to open the popover.

**To Reproduce**

**Result**

**Expected Result**

**Screenshots**

https://github.com/user-attachments/assets/94f7fe4b-5c52-4aa7-8efe-9092ed95162f

**Additional context**

**Comment by a-arias** [comment]
**TESTS**

We have enough amount of tests coverage ✅

<img width="881" height="793" alt="Image" src="https://github.com/user-attachments/assets/7b5c68e6-c471-476b-950e-55056bb40061" />
